types/sinon/index.d.ts

```diff
@@ -283,8 +272,6 @@ declare namespace Sinon {
 
         // Methods
         restore(): void;
```

Comment:
Should the restore() method also be moved to SinonFakeXMLHttpRequestStatic?

Reply:
@UselessPickles you would be correct it seems. this was leftover from what we had before, which was also wrong back then! 

looks to me like sinon's fake XHR has `restore` as a static method, _not_ as an instance method, so this doesn't belong here at all. it belongs only on the static interface

Reply:
Good to know I'm not insane :). This caused compiler errors for me, but I wasn't confident enough to submit my own PR for it.

Reply:
@UselessPickles  sorry if you had started on that, i just threw one up at #26119